extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -519,19 +509,18 @@ export class InstrumentationService {
    * @return {boolean} True if the trigger is allowed. False otherwise.
    */
   isTriggerAllowed_(triggerType, element) {
-    if (element.ownerDocument.defaultView != this.win_) {
+    if (element.ownerDocument.defaultView != this.ampdoc.win) {
       return ALLOWED_IN_EMBED.indexOf(triggerType) > -1;
     }
     return true;
   }
 }
 
 /**
- * @param {!Window} window
+ * @param {!Node|!../../../src/service/ampdoc-impl.AmpDoc} nodeOrDoc
  * @return {!InstrumentationService}
  */
-export function instrumentationServiceFor(window) {
-  return fromClass(window, 'amp-analytics-instrumentation',
+export function instrumentationServiceForDoc(nodeOrDoc) {
+  return fromClassForDoc(nodeOrDoc, 'amp-analytics-instrumentation',
```

Comment:
As mentioned above, use `getExistingServiceForDoc` instead. It's very dangerous to install a service twice.